examples/step-50/step-50.cc

```diff
@@ -1318,7 +1407,7 @@ void LaplaceProblem<dim, degree>::refine_grid()
 
   const double refinement_fraction = 1. / (std::pow(2.0, dim) - 1.);
   parallel::distributed::GridRefinement::refine_and_coarsen_fixed_number(
-    triangulation, estimate_vector, refinement_fraction, 0.0);
+    triangulation, error_estimator, refinement_fraction, 0.0);
 
```

Comment:
I find the name "estimator" confusing for a vector. Do you mind some other name?

Reply:
Hm, not sure here. We don't name the solution vector `solution_vector`, but just `solution`, and we use `system_rhs`. What I wanted to reflect is that we're estimating the *error* in the solution.

In step-6, we use `estimated_error_per_cell`. Would that be better?

Reply:
``error_estimate`` or ``estimated_error_per_cell`` sounds good